student.rb

```diff
@@ -1,15 +1,40 @@
+require_relative 'classroom'
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :name, :age, :parent_permission, :classroom
+
```

Comment:
- [ ] It would be nice if you make the classroom just readable and not accessible because it should be updated implicitly and by the app only but not externally :thinking:

Kindly make your getters and setters like below :arrow_down:

```ruby
attr_accessor :name, :age, :parent_permission
attr_reader :classroom
```